Lombiq.HelpfulLibraries.AspNetCore/Security/CdnContentSecurityPolicyProvider.cs

```diff
@@ -32,6 +32,7 @@ public class CdnContentSecurityPolicyProvider : IContentSecurityPolicyProvider
     public static ConcurrentBag<Uri> PermittedScriptSources { get; } = new(new[]
     {
         new Uri("https://cdn.jsdelivr.net/npm"),
+        new Uri("https://code.jquery.com/jquery-3.7.0.js"),
```

Comment:
Why include just this one JS and not the whole of https://code.jquery.com? We'd need to keep this up-to-date with OC versions (and the current one there is actually 3.7.1).

Reply:
Actually [only the host names](https://github.com/Lombiq/Helpful-Libraries/blob/09defab4fe25912372bf3a12e85b6136f3790f78/Lombiq.HelpfulLibraries.AspNetCore/Security/CdnContentSecurityPolicyProvider.cs#L85) of these URLs matter. So this is the same as
```suggestion
        new Uri("https://code.jquery.com/"),
```
I can remove the `jquery-3.7.0.js` if you think it's confusing.

Reply:
It is, yes, please.